### ISSUE TITLE
Linking stage build failure on OpenBSD 6.1

### ISSUE THREAD

**Issue by ismaell** [axel-download-accelerator/axel#95]
...
	gcc -std=gnu99  -Os -Wall -Wall  -lintl -pthread -o axel axel.o conf.o conn.o  ftp.o http.o search.o ssl.o  tcp.o text.o  -lssl
	/usr/bin/ld: cannot find -lintl
	collect2: ld returned 1 exit status
	*** Error 1 in src (Makefile:364 'axel')
	*** Error 1 in . (Makefile:485 'all-recursive')
	*** Error 1 in /home/ismael/axel (Makefile:358 'all')

**Comment by ordex** [comment]
Probably something like this should be enough if that option is not available on BSD:

```
diff --git a/configure.ac b/configure.ac
index 7606efd..855ff00 100644
--- a/configure.ac
+++ b/configure.ac
@@ -69,7 +69,7 @@ Darwin )
     LDFLAGS="$LDFLAGS -lintl"
 ;;
 FreeBSD | OpenBSD | NetBSD )
-    LDFLAGS="$LDFLAGS -lintl -pthread"
+    LDFLAGS="$LDFLAGS -pthread"
 ;;
 Linux | GNU/kFreeBSD | GNU )
     LDFLAGS="$LDFLAGS -pthread"

```

**Comment by ismaell** [comment]
I am not seeing any system providing linintl...

Am 30. August 2017 12:33:46 GMT-03:00 schrieb Antonio Quartulli <notifications@github.com>:
>Probably something like this should be enough if that option is not
>available on BSD:
>
>```
>diff --git a/configure.ac b/configure.ac
>index 7606efd..855ff00 100644
>--- a/configure.ac
>+++ b/configure.ac
>@@ -69,7 +69,7 @@ Darwin )
>     LDFLAGS="$LDFLAGS -lintl"
> ;;
> FreeBSD | OpenBSD | NetBSD )
>-    LDFLAGS="$LDFLAGS -lintl -pthread"
>+    LDFLAGS="$LDFLAGS -pthread"
> ;;
> Linux | GNU/kFreeBSD | GNU )
>     LDFLAGS="$LDFLAGS -pthread"
>
>```

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by ordex** [comment]
on macOS it builds with -lintl. But what do we need it for exactly?

**Comment by ismaell** [comment]
Isn't it part of gettext?
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by ismaell** [comment]
We need to add `-L/usr/local/lib` to `LDFLAGS`.

**Comment by ordex** [comment]
to which platform? Usually /usr/local/lib i used for manually installed libraries, no? standard installation should always copy libraries in /usr/lib*
If the user has the need to use -L/usr/local/lib then it should be added by him. no?

**Comment by ismaell** [comment]
That's not the case for *BSD.
Anyway, the true problem is that `configure.ac` is not using the output of `AM_GNU_GETTEXT`.

**Comment by ismaell** [comment]
Fixed by 968990414792e6c388028bd8d3aeff0a5768cf28.
